cli/src/cli.rs

```diff
@@ -82,6 +85,27 @@ where
                         println!("[error] {}\n", e);
                     }
                 },
+                Command::ExecuteFromFile(filename) => {
+                    if let Err(e) = self.load(&filename) {
+                        println!("[error] {}\n", e);
+                    }
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    pub fn load(&mut self, filename: &str) -> Result<()> {
+        let mut sqls = String::new();
+        File::open(filename)?.read_to_string(&mut sqls)?;
```

Comment:
<https://doc.rust-lang.org/stable/std/fs/struct.File.html#method.open>

[nit-picking]
How about using `AsRef<Path>` for extensibility?

Reply:
@ever0de 
Wow, you mean that replace `filename` type to `AsRef<Path>`?
```rust
pub fn load<P: AsRef<Path>>(&mut self, filename: P) -> Result<()> { 
    ..
```
Nice comment, thanks 👍

Reply:
Ok!